dash_bio/pileup.css

```diff
@@ -0,0 +1,300 @@
+/**
```

Comment:
Yeah -- not sure how I feel about this being in the `dash_bio` folder. Would be nice to include it in a `styles` folder within `src/lib/` or something similar.

Reply:
@shammamah Since `pileup.js` is used by a React component (namely, GenomeViewer), I can see why you would welcome a `styles/` subfolder in `src/lib/`, but I thought this CSS file might as well live in the (generically named) `assets/` folder at the root of the repo; hence 60464a5998909963a657366f518a2f618f859334.

Reply:
@shammamah oops! Since `assets/` lives outside of the `dash_bio` package, this creates a [dependency issue](https://circleci.com/gh/plotly/dash-bio/2394), so I included a `styles/` folder within the `dash_bio` package: 2dd37c706a126e9d4d88b15ca889163ff49a189b. I'm happy to rebase before merging!